### PR TITLE
Feature: Schema info for Pointcloud

### PR DESCRIPTION
**Pull request by haitheredavid**
## Description & motivation

Clouds are neat and it would be super neat to be able to send them to a stream in gh. 

## Changes:

- Added a `SchemaInfo` attribute

### CODE REVIEW

File: Objects/Objects/Geometry/Pointcloud.cs [hunk 1]
```diff
@@ -27,9 +27,18 @@ public class Pointcloud : Base, IHasBoundingBox, ITransformable<Pointcloud>
     public string units { get; set; }
 
     public Pointcloud()
+    
+    /// <param name="points">Flat list of x,y,z coordinates</param>
+    /// <param name="colors">Optional list of colors</param>
+    /// <param name="sizes">Optional list of sizes</param>
+    [SchemaInfo(nameof(Pointcloud), "Create a point cloud object", "Objects", "Geometry")]
+    public Pointcloud(List<double> points, List<int> colors = null, List<double> sizes = null)
```
Comment (teocomi) [inline Objects/Objects/Geometry/Pointcloud.cs:35]: Seems like `colors` is not being used?

**Comment by teocomi** [comment]
Hey David, isn't it already possible to use the native GH/Rhino point cloud objects instead?

**Comment by haitheredavid** [comment]
@teocomi rhino has this feature supported but there is no native grasshopper component that builds point clouds.

File: Objects/Objects/Geometry/Pointcloud.cs (hunk 1 above)
Comment (haitheredavid) [reply Objects/Objects/Geometry/Pointcloud.cs:37]: ah good catch, didn't see those lines missing. just pushed the fix

**Pull request merged**